mu-plugins/blocks/global-header-footer/blocks.php

```diff
@@ -973,6 +973,9 @@ function fetch_global_styles() {
 function set_current_item_class( $menu_items ) {
 	$current_url = get_menu_url_for_current_page( $menu_items );
 
+	// The download menu item is a button, we don't highlight it
+	if ( get_download_url() === $current_url ) return $menu_items;
```

Comment:
Coding standards would be to not have one-liners, despite it being exactly what I suggested :D 

```suggestion
	if ( get_download_url() === $current_url ) {
		return $menu_items;
	}
```